test/functional/Java12andUp/src/org/openj9/test/java_lang_constant/Test_DynamicConstantDesc.java

```diff
@@ -45,14 +50,77 @@
 	 * Test Java 12 API DynamicConstantDesc.resolveConstantDesc()
 	 */
 	@Test(groups = { "level.sanity" })
-	public void testDynamicConstantDescResolveConstantDesc() throws Throwable {
+	public void testDynamicConstantDescResolveConstantDescPrimitive() throws Throwable {
 		/* describe and resolve constant */
 		DynamicConstantDesc desc = DynamicConstantDesc.ofNamed(ConstantDescs.BSM_PRIMITIVE_CLASS, "I",
 				ConstantDescs.CD_Class, new ConstantDesc[0]);
 		Class<?> resolvedClass = (Class<?>)desc.resolveConstantDesc(MethodHandles.lookup());
+		DynamicConstantDesc resDesc = (DynamicConstantDesc)resolvedClass.describeConstable().orElseThrow();
 
 		/* verify that constant was resolve properly */
 		logger.debug("testDynamicConstantDescResolveConstantDesc: resolved class is: " + resolvedClass.toString());
 		Assert.assertTrue(resolvedClass.equals(int.class));
+
+		/* test equals */
+		logger.debug("original is: " + desc.toString() + " resolved is: " + resDesc.toString());
+		Assert.assertTrue(desc.equals(resDesc));
+	}
+
+	/* helpers for VarHandle tests */
+	private static ClassDesc helperClass = ClassDesc.of("org.openj9.test.java_lang_constant", "Test_DynamicConstantDesc")
```

Comment:
The formatting looks off here.  Better to avoid heavy levels of indenting by choosing a different point to break the line or just use a long line.

```java
private static ClassDesc helperClass = 
	ClassDesc.of("org.openj9.test.java_lang_constant", "Test_DynamicConstantDesc").nested("NestedHelper")
```